test/requests/code_action_resolve_expectations_test.rb

```diff
@@ -4,69 +4,71 @@
 require "test_helper"
 require "expectations/expectations_test_runner"
 
-class CodeActionResolveExpectationsTest < ExpectationsTestRunner
-  expectations_tests RubyLsp::Requests::CodeActionResolve, "code_action_resolve"
+module RubyLsp
```

Comment:
👍 I always prefer namespacing tests like this. I think it would have been preferable as as separate PR, but no need to change now.